.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -140,6 +140,12 @@ on:
         type: string
         default: 'false'
         required: False
+      enable_unitrace:
```

Comment:
is this option only for benchmarks? if so, I guess you can rename with prefix `benchmark_` here